### DISCUSSION TITLE
用本项目一键部署在huggingface上的模型能像vercel那样自动更新重部署吗

### DISCUSSION THREAD

**Discussion by peace000** [GaiZhenbiao/ChuanhuChatGPT#833] [Q&A]
还是每次项目更新都要手工重新部署个新模型？

**Comment by aqahwsp** [comment]
现在是每次都要手动更新，他那个huggingface上的版本太老了好久都没更了，现在要自己手动上传最新版的这个模型到huggingface的space中（上传前记得删掉.gitignore文件），上传后之后按那个老版本的配置好README.md文件（记得改一下sdk的版本，根据requirement里的文件来改），之后更改mouldes文件夹中config.py文件中的代码，固定设置server_name = "0.0.0.0"。这样才能在huggingface上顺利运行。
